pkg/importer/dataStream.go

```diff
@@ -320,19 +320,22 @@ func ResizeImage(dest, imageSize string) error {
 	if err != nil {
 		return err
 	}
-	currentImageSizeQuantity := resource.NewQuantity(info.VirtualSize, resource.BinarySI)
-	newImageSizeQuantity := resource.MustParse(imageSize)
-	minSizeQuantity := util.MinQuantity(resource.NewScaledQuantity(util.GetAvailableSpace(dest), 0), &newImageSizeQuantity)
-	if minSizeQuantity.Cmp(newImageSizeQuantity) != 0 {
-		// Available dest space is smaller than the size we want to resize to
-		glog.Warningf("Available space less than requested size, resizing image to available space %s.\n", minSizeQuantity.String())
-	}
-	if currentImageSizeQuantity.Cmp(minSizeQuantity) == 0 {
-		glog.V(1).Infof("No need to resize image. Requested size: %s, Image size: %d.\n", imageSize, info.VirtualSize)
-		return nil
+	if imageSize != "" && info != nil {
+		currentImageSizeQuantity := resource.NewScaledQuantity(info.VirtualSize, 0)
+		newImageSizeQuantity := resource.MustParse(imageSize)
+		minSizeQuantity := util.MinQuantity(resource.NewScaledQuantity(util.GetAvailableSpace(dest), 0), &newImageSizeQuantity)
+		if minSizeQuantity.Cmp(newImageSizeQuantity) != 0 {
+			// Available dest space is smaller than the size we want to resize to
+			glog.Warningf("Available space less than requested size, resizing image to available space %s.\n", minSizeQuantity.String())
+		}
+		if currentImageSizeQuantity.Cmp(minSizeQuantity) == 0 {
+			glog.V(1).Infof("No need to resize image. Requested size: %s, Image size: %d.\n", imageSize, info.VirtualSize)
+			return nil
+		}
+		glog.V(1).Infof("Expanding image size to: %s\n", minSizeQuantity.String())
+		return qemuOperations.Resize(dest, minSizeQuantity)
 	}
-	glog.V(1).Infof("Expanding image size to: %s\n", minSizeQuantity.String())
-	return qemuOperations.Resize(dest, minSizeQuantity.String())
+	return errors.New("Image resize called with blank resize")
```

Comment:
Or if "info == nil", if that's a possible thing

Reply:
its not